e2e/tests/session_steps_test.go

```diff
@@ -42,7 +42,21 @@ const (
 )
 
 func (s *suite) TheUserShouldWaitForTheModuleMessageToBeSubmitted(module, msgType string) {
-	s.waitForTxResultEvent(newEventMsgTypeMatchFn(module, msgType))
+	event := s.waitForTxResultEvent(newEventMsgTypeMatchFn(module, msgType))
+
+	// Currently the proof submission fee may be higher than the reward amount,
```

Comment:
NIT: Please review and reword sentences 2-4. A bit hard to understand (see my following comment first though)